contrib/create-static-manifest.sh

```diff
@@ -35,4 +35,5 @@ for filepath in $TEMPLATE_FILE; do
     # Remove any  files that are exactly 4 characters, the are "empty" files, we expect that these
     # files will only contain "---\n"
     find $OUTPUT_DIR -size 4c -delete
+    find $OUTPUT_DIR -size 0c -delete
```

Comment:
What does this do?

Reply:
delete files of size 0, there were are bunch of optional templates that cause empty files. This removes those

Reply:
Would you mind adding a comment please in a follow-up PR?